src/transformers/onnx/config.py

```diff
@@ -295,6 +309,17 @@ def generate_dummy_inputs(
             )
             # Generate dummy inputs according to compute batch and sequence
             dummy_input = [" ".join([preprocessor.unk_token]) * seq_length] * batch_size
+            if self.task == "multiple-choice":
+                # If dynamic axis (-1) we forward with a fixed dimension of 4 candidate answers to avoid optimizations
+                # made by ONNX
+                num_choices = compute_effective_axis_dimension(
+                    num_choices, fixed_dimension=OnnxConfig.default_fixed_num_choices, num_token_to_add=0
+                )
+                dummy_input = dummy_input * num_choices
+                tokenized_input = preprocessor(dummy_input, text_pair=dummy_input)
```

Comment:
I would just add a small comment to make clear what's happening here.
What do you think?

```suggestion
                # Shape is [batch_size x num_choices, seq_length]
                tokenized_input = preprocessor(dummy_input, text_pair=dummy_input)
```